### PR TITLE
Added MIT License

### LINKED ISSUE #1: Add MIT license

### PR DESCRIPTION
**Pull request by ajaygargdev**
fixes #1

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2019 KMD A/S
```
Comment (adamchester) [inline LICENSE.md]: 2020?

Comment (ajaygargdev) [reply LICENSE.md]: Yes, Fixed

**Review by adamchester — APPROVED** [review]

**Comment by adamchester** [comment]
Build failed because branch name has underscores

**Comment by ajaygargdev** [comment]
Should we create new branch?

**Pull request merged**